src/lib/viewers/controls/color-picker/__tests__/ColorPickerControl-test.tsx

```diff
@@ -1,39 +1,31 @@
 import * as React from 'react';
 import { shallow, ShallowWrapper } from 'enzyme';
-import { AnnotationMode } from '../../annotations/types';
+import { bdlBoxBlue } from 'box-ui-elements/es/styles/variables';
 import ColorPickerControl from '../ColorPickerControl';
 
 describe('ColorPickerControl', () => {
-    const onAnnotationColorClick = jest.fn();
-
     const getWrapper = (props = {}): ShallowWrapper =>
-        shallow(<ColorPickerControl onAnnotationColorClick={onAnnotationColorClick} {...props} />);
+        shallow(<ColorPickerControl colors={[bdlBoxBlue]} onColorSelect={jest.fn()} {...props} />);
 
     const getToggleButton = (wrapper: ShallowWrapper): ShallowWrapper => wrapper.find('.bp-ColorPickerControl-button');
 
     describe('render', () => {
```

Comment:
Should there be a test for the `onSelect` callback?

Reply:
Yes we should test that.